react/features/recording/components/web/RecordingLabel.tsx

```diff
@@ -38,16 +38,29 @@ class RecordingLabel extends AbstractRecordingLabel {
      * @inheritdoc
      */
     _renderLabel() {
-        if (this.props._status !== JitsiRecordingConstants.status.ON) {
-            // Since there are no expanded labels on web, we only render this
-            // label when the recording status is ON.
+        const { _isTranscribing, _status, classes, mode, t } = this.props;
+        const isRecording = mode === JitsiRecordingConstants.mode.FILE;
+        const icon = isRecording || _isTranscribing ? IconRecord : IconSites;
+        const isOn = _status === JitsiRecordingConstants.status.ON;
+        let content;
+
+        if (!isOn && !_isTranscribing) {
```

Comment:
logic seems a bit convoluted, seems like it would be easier to follow it if we just return for `!isOn` separately and remove it from subsequent checks, unless there's a case where we do display something when `isOn` is `false` (which could be the last `else` block but I'm not sure if that was the intent)